fn_err.ri 1,14: `i` is duplicated
  > i
    ^
fn_err.ri 2,10: unknown type `foo`
  > foo
    ^^^
fn_err.ri 5,11: mismatched types: `string` (expected) and `int` (found)
  > i
    ^
fn_err.ri 7,5: `g` is duplicated
  > g
    ^
fn_err.ri 10,2: `v` is not function (`int`)
  > v()
    ^^^
fn_err.ri 12,2: cannot call `int`
  > 0()
    ^^^
